### PR TITLE
Code of Conduct Committee Elections — 2025

### PR DESCRIPTION
**Pull request by justaugustus**
*In an effort to deconflict between Steering and Code of Conduct Committee
election timing and provide new Steering Committee members sufficient time to
onboard (3+ months), the Steering Committee has made the decision to move the
Code of Conduct Committee elections to follow the 2024 Steering elections,
instead of preceding them.*

I am proposing the following timeline for the 2025 Code of Conduct Committee elections:

| Date | Event |
| --- | --- |
| Monday, December 2 | Nominations Open |
| Friday, January 31 | Nominations Close |
| Monday, February 3 | Steering to confirm candidacy |
| Monday, February 10 | Election Begins |
| Monday, February 17 | Election Closes |
| Tuesday, February 18 | Announcement of results |
| Thursday, February 20 | CoCC to hold onboarding sessions for new members |

As a point of reference, the last CoCC election that we held had approximately the following timeline:

- Nominations period: 2 weeks
- Steering confirms candidacy: 1 day after nomination close
- Election begins: 1 week after candidacy confirmation
- Election closes: 1 week after election opens
- Announcement: 1 day after election closes
- Onboarding: 3 days after election announcement

tl;dr — the biggest changes here are:

- the actual election dates
- how long the nominations period is open for (2 months, instead of 2 weeks, to accommodate holiday schedules and multiple notifications)

### CODE REVIEW

File: elections/code-of-conduct/2025/election.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: 2025 Kubernetes Code of Conduct Committee Election
+organization: Kubernetes
+# TODO(cocc): Fix election dates
+start_datetime: 2025-08-11 00:00:01
```
Comment (jberkus) [inline elections/code-of-conduct/2025/election.yaml:4]: If the elections aren't until August, why are we doing this now?

Comment (justaugustus) [reply elections/code-of-conduct/2025/election.yaml:4]: @jberkus — We will be proposing a February election timeline to the CoCC.

Please note the WIP tag, the [`/hold` comment](https://github.com/kubernetes/community/pull/8137#issuecomment-2460433608), and the TODOs throughout the PR.

This is not yet ready for review.

**Comment by salaxander** [comment]
@justaugustus The timeline overall looks good to me. I'd also be comfortable shortening the nomination window to `01/06/25 - 01/31/25`, but there's no strong reason that it can't be the full 2 months.

**Comment by BenTheElder** [comment]
LGTM, thank you!

File: elections/code-of-conduct/2025/results.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Results of the 2025 Kubernetes Code of Conduct Committee Elections
+
+- Number of seats open: 3 (2 year term)
+- Number of eligible voters: 7
+- Number of votes cast: TBD
+- Turnout: TBD
```
Comment (BenTheElder) [inline elections/code-of-conduct/2025/results.md:6]: ... can you imagine if it's not 100%? 😱

**Comment by AnaMMedina21** [comment]
New dates look good to me 👍🏽

**Comment by justaugustus** [comment]
Sweet! We've got quorum for CoCC!

Once the rest of Steering checks in, I'll update the dates throughout.

**Comment by endocrimes** [comment]
Timeline also lgtm

File: elections/code-of-conduct/2025/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+The role of this election is to fill out the three (3) seats open for election
+this year on the [Kubernetes Code of Conduct Committee]. Each elected
+member will serve a two (2) year term.
+
+<!-- TODO(cocc): Add blurb about election timing -->
+
+## Election Platform
+
+We will be using [Elekto] to conduct the election. Elekto relies entirely on GitHub Oauth for
```
Comment (soltysh) [inline elections/code-of-conduct/2025/README.md:32]:
```suggestion
We will be using [Elekto] to conduct the election. Elekto relies entirely on GitHub OAuth for
```

**Review by soltysh — COMMENTED** [review]
the initial timeline lgtm, the rest of the PR still needs updates

**Review by justaugustus — COMMENTED** [review]
We have unanimity from @kubernetes/code-of-conduct-committee on the proposed timeline and a quorum from Steering.

I've updated dates throughout the PR and released the hold.

@kubernetes/steering-committee — needs re-lgtm for the merge.

**Comment by justaugustus** [comment]
(actually releasing the hold)
/hold cancel

**Comment by BenTheElder** [comment]
/lgtm
Thanks all!
